test/unit/environment.test.js

```diff
@@ -87,14 +87,12 @@ describe('the environment scraper', function () {
     }).not.throws()
   })
 
```

Comment:
There's still a reference to the async library in this test file (the import to the `a` constant)...though since `async` is still being used in package.json for the moment, this should be ok.

Reply:
my ticket focused on agent code not tests.  weren't you working on this?

Reply:
Yes--I'm doing conversion to tap + removing the async from the test here. (I'll remove the conflicts in my branch.)

Reply:
My initial reading was that this would remove the async lib as a dependency, which would cause a problem if this test was still referencing it.

Reply:
oh sorry. i moved it from a dependency to a dev dependency